homeassistant/components/caldav/calendar.py

```diff
@@ -55,6 +56,7 @@
             ],
         ),
         vol.Optional(CONF_VERIFY_SSL, default=True): cv.boolean,
+        vol.Optional(CONF_PERIOD, default=1): cv.positive_int,
```

Comment:
I don't think we should call this period if it's limited to days. Either we should call it something with `days` or expand the feature to allow a time period by using `cv.time_period`.

Reply:
no problem, I just changed "period" to "days" and the documentation accordingly :)